Client/Teztez/Core/UIComponents/CommonButton/CommonButton.swift

```diff
@@ -0,0 +1,28 @@
+//
+//  CommonButton.swift
+//  Teztez
+//
+//  Created by Adlet on 4/22/20.
+//  Copyright © 2020 crococoders. All rights reserved.
+//
+
+import UIKit
+
+final class CommonButton: UIButton {
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        commonInit()
+    }
+
+    required init?(coder: NSCoder) {
+        super.init(coder: coder)
+        commonInit()
+    }
+
+    private func commonInit() {
```

Comment:
Oh-oh, some kaspi boys in da house :)
I opt for using something simplistic like `setup` or `setupUI`
What do you think @zainoldin ?

Reply:
I think setup is ok